tools/generate-bidi-types.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+outfile=src/protocol/webdriver-bidi.ts
```

Comment:
```suggestion
readonly OUTFILE="src/protocol/webdriver-bidi.ts"
```

https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Please update `outfile -> OUTFILE` below as well.